packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.scss

```diff
@@ -7,17 +7,62 @@
 // --------------------------------------------------
 // Search box styles
 
-
-$SearchBox-field-padding-left: 10px;
-$SearchBox-height: 36px;
-
 .root {
   @include ms-font-m;
   @include ms-u-normalize;
   color: $ms-color-neutralPrimary;
   position: relative;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  align-items: stretch;
   margin-bottom: 10px;
   border: 1px solid $ms-color-themeTertiary;
+  height: 32px;
+}
+
+.icon {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  flex-shrink: 0;
+  color: $ms-color-neutralSecondaryAlt;
+  font-size: 16px;
+  flex-basis: 32px;
+  text-align: right;
+  transition: flex-basis $ms-duration1;
+  color: $ms-color-themePrimary
+}
+
+// Input Field
+.field {
+  @include ms-u-normalize;
+  @include padding(0px, 0, 1px, 8px);
```

Comment:
yeah.. i need to take a look at all of the inputs and how they achieve their baselines. This 1px was needed to make it match with other designs. A pass over all inputs might be able to find a way to create them all consistently and avoid yucky hacks like that.